assets/facets.js

```diff
@@ -62,6 +62,8 @@ class FacetFiltersForm extends HTMLElement {
         FacetFiltersForm.renderFilters(html, event);
         FacetFiltersForm.renderProductGridContainer(html);
         FacetFiltersForm.renderProductCount(html);
+        var api = new Yotpo.API(yotpo);
+        api.refreshWidgets();
```

Comment:
just curious, do we need this here? is it enough to just call `api.refreshWidgets()` once below on line 75? Or do we have to call it for each section here?

Reply:
Yeah, the `api.refreshWidgets()` needs to be called for each static method. Should I make the `var api = new Yotpo.API(yotpo)` a global scope variable though (outside of the class constructor)?